build-image/builder.sh

```diff
@@ -53,9 +61,15 @@ for platform in ${GOLANG_CROSSPLATFORMS}; do
     mv ./${product} /tmp/go-build/${product}/${product}
   fi
 
-  if [ -e .artifact_files ]; then
+  if [ -e /tmp/go-build/.artifact_files ]; then
```

Comment:
it may be advantageous to use `mktemp` as to be able to run multiple builds in parallel.